bindings/python/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "iota-sdk-python"
-version = "0.1.0"
+version = "1.0.0-rc.1"
```

Comment:
We started nodejs with rc.0

Reply:
Yes but that seemed weird to me, https://semver.org/spec/v2.0.0-rc.1.html starts with 1 :thinking:
But this doesn't seem to be defined anywhere, so if you want I can change to 0

Reply:
Updated